lib/Os.php

```diff
@@ -44,45 +47,59 @@ public function __construct() {
 	/**
 	 * @return bool
 	 */
-	public function supported() {
+	public function supported(): bool {
 		$data = $this->backend->supported();
 		return $data;
 	}
 
 	/**
 	 * @return string
 	 */
-	public function getHostname() {
+	public function getHostname(): string {
 		return (string)gethostname();
 	}
 
 	/**
+	 * Get name of the operating system.
+	 *
 	 * @return string
 	 */
 	public function getOSName(): string {
 		return PHP_OS . ' ' . php_uname('r') . ' ' . php_uname('m');
 	}
 
+	/**
+	 * @return Memory
+	 */
 	public function getMemory(): Memory {
 		return $this->backend->getMemory();
 	}
 
+	/**
+	 * @return string
+	 */
```

Comment:
when we have return type hints, we don't need phpdoc just for this. not a blocker, for next time.